core-ui/src/shared/components/WorkloadSelector/WorkloadSelector.js

```diff
@@ -25,10 +22,7 @@ export const WorkloadSelector = ({ resource, labels }) => {
   return (
     <LayoutPanel className="fd-margin--md" key="workload-selector">
       <LayoutPanel.Header>
-        <LayoutPanel.Head
-          title={t('workload-selector.title')}
-          className="header"
-        />
+        <LayoutPanel.Head title={title} className="header" />
```

Comment:
Let's add a default title
```suggestion
        <LayoutPanel.Head title={title || t('workload-selector.title')} className="header" />
```